worlds/tunic/er_data.py

```diff
@@ -614,37 +626,40 @@ class DeadEnd(IntEnum):
     "Beneath the Well Back": RegionInfo("Sewer"),  # the back two portals, and all 4 upper chests
     "West Garden": RegionInfo("Archipelagos Redux"),
     "Magic Dagger House": RegionInfo("archipelagos_house", dead_end=DeadEnd.all_cats),
-    "West Garden Portal": RegionInfo("Archipelagos Redux", dead_end=DeadEnd.restricted),
+    "West Garden Portal": RegionInfo("Archipelagos Redux", dead_end=DeadEnd.restricted, outlet_region="West Garden by Portal"),
```

Comment:
Would it be worth considering making `outlet_region` an enum? 
eh, after looking at this more it seems like it probably wouldn't since you'd be essentially converting each region name to an enum value. Disregard 😅